src/components/App.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+
+import "../styles/App.css";
+import getForecast from "../requests/getForecast";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetails from "./ForecastDetails";
+import SearchForm from "./SearchForm";
+
+function App() {
+  const [forecasts, setForecasts] = useState([]);
+  const [location, setLocation] = useState({ city: "", country: "" });
+  const [selectedDate, setSelectedDate] = useState(0);
+  const [searchText, setSearchText] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const selectedForecast = forecasts.find(
+    (forecast) => forecast.date === selectedDate
+  );
+  useEffect(() => {
```

Comment:
Its a common practice to leave a clear line before and after hook calls like this, just makes the file easier to read at a glance:
```
  );

  useEffect(() => {
```